pymobiledevice3/exceptions.py

```diff
@@ -141,6 +141,11 @@ class ConnectionTerminatedError(PyMobileDevice3Exception):
     pass
 
 
+class StreamClosedError(ConnectionTerminatedError):
+    """ Raise trying to send a message on a closed stream. """
```

Comment:
Raise when trying to send a message on a closed stream?

Reply:
fixed